.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v2
+      - uses: olafurpg/setup-gpg@v2
+      - name: Publish
+        run: csbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }} # Running as coltfred
```

Comment:
Thanks for adding this comment, but please don't run CI/CD as coltfred. You should be able to copy credentials from https://github.com/IronCoreLabs/ironoxide-java/tree/master/.github instead.

Reply:
This is an open source project which is currently running as me, after we move it to github if we want to migrate the user, keys etc we can. For this task I'm trying to change as little as possible.